content/applications/sales/members.rst

```diff
@@ -0,0 +1,108 @@
+=======
+Members
+=======
+
+The **Members** application is where all operations related to memberships can be configured and
+managed. The **Members** app integrates with the **Sales** and **Accounting** applications to sell
+and invoice memberships directly to customers.
+
+Membership products
+===================
+
+To create a new membership product, navigate to :menuselection:`Members app --> Configuration -->
+Membership Products`, and click :guilabel:`New` to open a blank product record.
+
+On the new product form, enter a name for the product in the field under :guilabel:`Product Name`.
+Use the two blank fields to the right of :guilabel:`Membership Duration` to set a date from which
+membership becomes active. Finally, use the :guilabel:`Membership Fee` field to set the price at
+which the product is sold to customers.
+
+The following optional fields can also be configured for each product:
+
+An internal reference number can be set in the field to the right of :guilabel:`Internal Reference`,
+and a category set using the dropdown to the right of :guilabel:`Product Category`. Set a company
+using the field to the right of :guilabel:`Company` for the product to appear in that company's
+inventory. Use the slider to the right of :guilabel:`Active` to set hide a product without removing
+it.
+
+Tick the checkbox next to :guilabel:`Visible on current website` if desired, and use the field
+under :guilabel:`Description` for an internal product description, and :guilabel:`Sales Description`
+for a product description that will be visible on quotations. Use the field to the right of
+:guilabel:`Income Account` to use a different value than the default from the product category.
+Finally, set the default tax used when selling this product using the field next to
+:guilabel:`Taxes`.
```

Comment:
While this info is correct, I feel like most docs with a section like this tend use a list format.

[name of field] : Explanation of the field.

That tends to be well received... so, consider that during your revisions.

The way it is currently written doesn't rely the information as cleanly (or as efficiently) as possible

Reply:
Upon further thought, I think you should *definitely* rework/reformat/reword this section so it is a cleaner, more digestible list. Doing so will add to the overall value of the doc. Thanks!